commands/scale/cluster/command_test.go

```diff
@@ -370,52 +425,97 @@ func TestScaleCluster(t *testing.T) {
 					]
 				}
 			}`))
-		} else {
-			w.WriteHeader(http.StatusNotFound)
-			w.Write([]byte(`{"code": "RESOURCE_NOT_FOUND", "message": "Could not find this."}`))
-		}
-	}))
-	defer mockServer.Close()
+				} else {
```

Comment:
is the indentation right? it is hard to judge in Github

Reply:
Yesss it is, check it out with the blocks collapsed
![image](https://user-images.githubusercontent.com/13508038/81267142-43555e00-9046-11ea-826a-92761312fe9f.png)